Framework/Algorithms/src/CreateGroupingWorkspace.cpp

```diff
@@ -70,11 +70,13 @@ void CreateGroupingWorkspace::init() {
                   "Use / or , to separate multiple groups. "
                   "If empty, then an empty GroupingWorkspace will be created.");
 
-  std::vector<std::string> grouping{"", "All", "Group", "Column", "bank"};
+  std::vector<std::string> grouping{"", "All", "Group", "2_4Grouping", "Column",
+                                    "bank"};
   declareProperty(
       "GroupDetectorsBy", "", boost::make_shared<StringListValidator>(grouping),
       "Only used if GroupNames is empty: All detectors as one group, Groups "
-      "(East,West for SNAP), Columns for SNAP, detector banks");
+      "(East,West for SNAP), 2_4Grouping for SNAP, Columns for SNAP, detector "
```

Comment:
The meaning of `2_4Grouping` is mentioned in the release notes, but not in the algorithm help